src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,19 @@
 package mate.academy.service;
 
 public class AuthenticationService {
+    private final UserService userService = new UserService();
     /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
+    * Imagine that some user wants to login to your site.
+    * You should check if user credentials (login and password) are valid or not.
+    * All users are stored in <code>UserService</code> class.
+    * @param email - user's email
+    * @param password - user's password
+    * @return true if user by email exists and passed password is equal to user's password.
+    * Return false in any other cases.
+    */
+
     public boolean login(String email, String password) {
-        return false;
+        return (userService.findByEmail(email) != null && password
+        .equals(userService.findByEmail(email).getPassword())) ? true : false;
```

Comment:
better create separate variable for  `userService.findByEmail(email)` and avoid redundant call of this method